Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/certificateManager.xhtml

```diff
@@ -348,7 +348,12 @@
                                             <p:autoUpdate />
                                         </p:messages>
                                         <div class="content">
-                                            <h:outputText>Select an option below to continue with Import Wizard.</h:outputText>
+                                            <h:outputText>This Import Wizard allows the user to add or import one certificate at a time.</h:outputText>
+                                            <br /> <br />
+                                            <h:outputText>The Import Wizard thru a couple of steps makes a certificate, sets up the Certificate Signing Request (CSR) and allows for the certificate to be sent to be processed by a CA provider (Entrust or eHealth Exchange). Once, the certificate has been verified by the CA provider, the Import Wizard can resume loading the Key and Trust Stores.</h:outputText>
```

Comment:
thru -> through
Once, the certificate has been verified by the -> Remove comma after "Once"